docs/quantum_oscilator/architecture.md

```diff
@@ -0,0 +1,81 @@
+# Architecture of Neutal Network
+
+## Parent classes
+
+[`#!python class tf.keras.Model`](https://keras.io/api/models/model/)
+
+## Introduction
+
+QONetwork class is responsible for the learning process from the user input.
+Our neural network architecture that simultaneously learns eigenvalues and the
```

Comment:
I have the impression that this sentence does not make much sense, please correct it - it is not architecture that learns eigenvalues etc.